pkg/services/ngalert/api/lotex_ruler_test.go

```diff
@@ -0,0 +1,135 @@
+package api
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"testing"
+
+	"github.com/grafana/grafana/pkg/infra/log"
```

Comment:
the formatter won't enforce this, but our preference for import formatting looks like:
```go
import (
  // stdlib packages
 "context"
 "errors"

  // non-grafana packages
  "github.com/stretchr/testify/require"
  
  // packages in the local repository
  "github.com/grafana/grafana/pkg/models"
  "github.com/grafana/grafana/pkg/web"
)
```

do you have an editor setting that's leaving these empty lines in-between packages?